File: Questions/Day-3/java/Q2.java
import java.util.Scanner;

public class Q2 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        int n = in.nextInt();

        int temp = n;
        int rem = 0, sum = 0;
        while (n > 0) {
            rem = n % 10;
            sum = (sum * 10) + rem;
            n = n / 10;
        }
        if (temp == sum)
            System.out.println("palindrome number ");
        else
            System.out.println("not palindrome");
    }
}

File: Questions/Day 3/cpp/Q2.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main()
{

    int n = 0;
    cin >> n;

    int temp = n;
    int rem = 0, sum = 0;
    while (n > 0)
    {
        rem = n % 10;
        sum = (sum * 10) + rem;
        n = n / 10;
    }
    if (temp == sum)
        cout << "palindrome number ";
    else
        cout << "not palindrome";

    return 0;
}